### PR TITLE
US754190: Release preparation

### PR DESCRIPTION
**Pull request by frankmccarry**

### CODE REVIEW

File: release-notes-3.3.2.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-
 #### Version Number
 ${version-number}
 
 #### New Features
 - None
 
 #### Bug Fixes
-739109: Fix for issue where truncated emails were not getting processed.
+739109: Fix for issue where truncated emails were not getting processed. This caused a processing error
```
Comment (michael-bryson) [inline release-notes-3.3.2.md]:
```suggestion
- 739109: Fix for issue where truncated emails were not getting processed.  
This caused a processing error to get added to the email.
```

**Review by michael-bryson — CHANGES_REQUESTED** [review]
How did you re-version this repo? The pom versions have changed to 3.3.1 but the release notes file is 3.3.2?

**Comment by frankmccarry** [comment]
> How did you re-version this repo? The pom versions have changed to 3.3.1 but the release notes file is 3.3.2?

Hi, I just noticed that as well and pushed change

**Comment by michael-bryson** [comment]
The release notes still do not look right if you view the file:
image

You need 2 spaces after the bullet point and then the next sentence on a new line as per my suggestion above.

**Comment by frankmccarry** [comment]
> The release notes still do not look right if you view the file: image
> 
> You need 2 spaces after the bullet point and then the next sentence on a new line as per my suggestion above.

I had not intended there to be a new line there, I'll add the new line after the full stop now. I do not see immediately how that is clearer, but if that's the standard will make the change.

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**